examples/js/loaders/TDSLoader.js

```diff
@@ -205,6 +205,12 @@
 
 			while ( next !== 0 ) {
 
+				if (next === 0x3338 ) {
```

Comment:
The loader does not inline chunk IDs but defines constants with names for better readability. 

When `0x3338` is no supported chunk ID, it seems your file is corrupt in some way. At least the exporter produced the file is buggy. I'm not sure we should add fallbacks for use cases like that,